Manual/HTML/Polygonal Mapping.html

```diff
@@ -0,0 +1,232 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1">
+    <title>Polygonal Mapping — Keep Talking and Nobody Explodes Module</title>
+    <link rel="stylesheet" type="text/css" href="css/font.css">
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <script src="js/ktane-utils.js"></script>
+    <script src="js/jquery.3.7.0.min.js"></script>
+    <script src="js/Utilities/svg-utils.js"></script>
+    <script>
+        const BigTable = [
+            [38, 18, 19, 17, 30, 8,  17, 26, 32, 26],
+            [43, 3,  10, 19, 17, 47, 20, 26, 29, 37],
+            [28, 3,  5,  26, 7,  28, 40, 14, 36, 16],
+            [32, 26, 37, 0,  1,  39, 34, 5,  42, 43],
+            [28, 36, 40, 19, 32, 35, 0,  12, 48, 43],
+            [9,  27, 5,  30, 14, 9,  29, 36, 48, 24],
+            [47, 22, 31, 2,  20, 12, 18, 23, 38, 10],
+            [6,  38, 48, 9,  27, 29, 9,  13, 46, 35],
+            [0,  22, 14, 48, 37, 10, 38, 3,  48, 23],
+            [39, 14, 46, 24, 47, 45, 30, 2,  29, 13],
+            [45, 40, 17, 35, 46, 42, 35, 28, 31, 13],
+            [33, 40, 11, 25, 21, 45, 45, 22, 16, 10],
+            [20, 14, 30, 1,  41, 12, 5,  47, 4,  39],
+            [33, 44, 32, 23, 11, 3,  0,  0,  27, 7 ],
+            [6,  20, 6,  34, 19, 30, 25, 31, 43, 35],
+            [1,  21, 2,  10, 19, 8,  11, 23, 34, 8 ],
+            [44, 41, 9,  42, 15, 4,  42, 3,  16, 13],
+            [11, 36, 14, 36, 1,  40, 1,  43, 37, 22],
+            [7,  24, 25, 2,  38, 39, 22, 7,  24, 24],
+            [8,  33, 47, 2,  33, 25, 21, 41, 12, 17],
+            [15, 15, 6,  42, 23, 31, 4,  12, 46, 18],
+            [41, 2,  46, 32, 5,  34, 41, 21, 18, 15],
+            [16, 25, 0,  13, 1,  37, 31, 27, 29, 33],
+            [11, 34, 20, 15, 9,  18, 10, 4,  3,  5 ],
+            [7,  8,  13, 44, 21, 12, 45, 39, 6,  7 ],
+            [28, 4,  27, 6,  4,  44, 11, 8,  16, 44]
+        ];
+        const imgIndexes = [
```

Comment:
I have no experience using JavaScript but it might be better practice to name these something specific like `polygonalMappingBigTable` so there will never be conflicts when implemented into the main manual